modules/ROOT/pages/faq.adoc

```diff
@@ -81,7 +81,11 @@ https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/[co
 * Twitter at https://twitter.com/fedora[@fedora] (all Fedora and
 other relevant news)
 
-https://discussion.fedoraproject.org/t/launch-faq-what-are-the-communication-channels-around-fedora-coreos/46/1[Discuss on discussion.fedoraproject.org]
+There is a community meeting that happens every Wednesday at 16:30 UTC.  See the https://apps.fedoraproject.org/calendar/CoreOS[Fedora CoreOS fedocal] for the most up-to-date information.
+
+If you think you have found a problem with Fedora CoreOS, file an issue in our https://github.com/coreos/fedora-coreos-tracker/issues[issue tracker].
+
+If you have a question or an idea you'd like to discuss, we have a our own https://discussion.fedoraproject.org/t/launch-faq-what-are-the-communication-channels-around-fedora-coreos/46/1[Discourse forum], too.
```

Comment:
Actually, every single section in this FAQ has a "Discuss on discussion.fedoraproject.org" link at the bottom, which leads to a separate discussion in there. I'm not fully sure why it is this way, but your patch departed from the "standard". So I'm just notifying you.

Reply:
Ah, I just blindly copied that link...I'll make another adjustment.  Thanks!